README.md

```diff
@@ -316,6 +333,80 @@ Use the `--trim` option to strip specific prefixes from the paths, to make the g
 
+See [Blame filtering](blame-filtering) below for how to use the `--blame-filter` option.
+
+# Blame filtering
+
+Use the `sarif blame` command to augment a SARIF file or multiple SARIF files with blame information.
+
+Blame information is added to the property bag of each `result` object for which it was successfully obtained.  The keys and values used are as in the [git blame porcelain format](https://git-scm.com/docs/git-blame#_the_porcelain_format).  E.g.:
+
+```json
+{
+  "ruleId": "SM00702",
+  ...
+  "properties": {
+    "blame": {
+      "author": "aperson",
+      "author-mail": "<aperson@acompany.com>",
+      "author-time": "1350899798",
+      "author-tz": "+0000",
+      "committer": "aperson",
+      "committer-mail": "<aperson@acompany.com>",
+      "committer-time": "1350899798",
+      "committer-tz": "+0000",
+      "summary": "blah blah commit comment blah",
+      "boundary": true,
+      "filename": "src/net/myproject/mypackage/MyClass.java"
+    }
+  }
+}
+```
+Note that the bare `boundary` key is given the automatic value `true`.
+
+This blame data can then be used for filtering and summarising via the `--blame-filter` option available for various commands.  This option requires a path to a filter-list file, containing a list of patterns and substrings to match against the blame information author email.  The format of a filter-list file is as follows:
+
+```
+# Lines beginning with # are interpreted as comments and ignored.
+# A line beginning with "description: " is interpreted as an optional description for the filter.  If no title is specified, the filter file name is used.
+description: Example filter from README.md
+# Lines beginning with "+: " are interpreted as inclusion substrings.  E.g. the following line includes issues whose author-mail field contains "@microsoft.com".
++: @microsoft.com
+# The "+: " can be omitted.
+@microsoft.com
+# Instead of a substring, a regular expression can be used, enclosed in "/" characters.  Issues whose author-mail field includes a string matching the regular expression are included.  Use ^ and $ to match the whole author-mail field.
++: /^<myname.*\.com>$/
+# Again, the "+: " can be omitted for a regular expression include pattern.
+/^<myname.*\.com>$/
+# Lines beginning with "-: " are interpreted as exclusion substrings.  E.g. the following line excludes issues whose author-mail field contains "@microsoft.com".
+-: @bad.microsoft.com
+# Instead of a substring, a regular expression can be used, enclosed in "/" characters.  Issues whose author-mail field includes a string matching the regular expression are excluded.  Use ^ and $ to match the whole author-mail field.  E.g. the following pattern excludes all issues whose author-mail field contains a GUID.
+-: /[0-9A-F]{8}[-][0-9A-F]{4}[-][0-9A-F]{4}[-][0-9A-F]{4}[-][0-9A-F]{12}/
+```
+
+Or, without the comments and alternative forms:
+
+```
+description: Example filter from README.md
```

Comment:
Might be good to have some more examples - or maybe give more detail in this description, like "Example filter to include all @microsoft.com addresses and exclude all @bad.microsoft.com addresses" (more like the description below for "Exclude everything filterable").

Reply:
I've reworded it a bit to explain what it does, also fixed up an incorrect descriptions in there :-)